GalilSup/Db/galil_motor_extras.template

```diff
@@ -36,6 +36,39 @@ record(scalcout, "$(P):$(M):MANAGERMODE")
     field(OOPT, "Every Time")
 }
 
+#Required for motor record 6-10 and up
+#Below records are used to extract data from MR
+#and pass to driver
+# Motor direction for this axis
+record(longout,"$(P):$(M)Direction") {
```

Comment:
These new records don't follow the naming convention we usually use (and use in the rest of this file). Can we change them, or if not, can we alias them to some more standard names as well.

Reply:
This addition is for compatibility with https://github.com/epics-modules/motor/blob/master/motorApp/Db/asyn_motor.db and is to make the updated motor record work properly as Galil uses its own equivalent of asyn_motor.db  I don't think there is any advantage in aliasing it, but I should move it to galil_motor.template instead (which is where it is in the new galil driver)